Cargo.toml

```diff
@@ -35,6 +36,8 @@ wasm-chain-integration = { version = "*", path = "./concordium-base/smart-contra
 
 [features]
 postgres = ["tokio-postgres"]
+# An internal feature used to generate Rust sources from protobuf schemas.
```

Comment:
Just a comment, I don't see it as an internal feature, users could enable for their local version of protobuf.
The `prometheus` crate does this.

Reply:
As discussed on slack, I really don't see a use for this.

They cannot really have their own local version of protobuf since all of the definitions are hidden. They are just used internally, and the external API of the SDK cannot be used to access any of the generated files. That is by design as it is now.